### PR TITLE
doc: import-in-the-middle release token

### PR DESCRIPTION
**Pull request by legendecas**
The requested token should have the following permissions:

```
permissions:
  contents: write
  pull-requests: write
```

https://github.com/googleapis/release-please-action?tab=readme-ov-file#workflow-permissions

/cc @mhdawson the repo has already a token configured with my personal token. please feel free to replace it with @nodejs-github-bot's one.

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

File: request-an-access-token.md [hunk 1]
```diff
@@ -34,9 +34,15 @@ The "secret name" is a string that the secret can be referenced in the GitHub Ac
...
-nodejs/import-in-the-middle | RELEASE_PLEASE_GITHUB_TOKEN
+The "expiration date" is the date before which the token should be renewed and
+replaced. This should be no longer than 1 year.
+
+The "pull request" is the PR that initially requested the token, or requested
+permission scope changes. The PR should describe the permission scopes requested.
+
+Repo                        | Secret name                  | Expiration date | Pull Request                              |
+---                         | ---                          | ---             | ---                                       |
+nodejs/import-in-the-middle | RELEASE_PLEASE_GITHUB_TOKEN  | 2025-07-31      | https://github.com/nodejs/admin/pull/902  |
```
Comment (mhdawson) [inline request-an-access-token.md:45]:
```suggestion
nodejs/import-in-the-middle | RELEASE_PLEASE_GITHUB_TOKEN  | 2025-07-23      | https://github.com/nodejs/admin/pull/902  |
```

**Comment by mhdawson** [comment]
Created the token as the original request was already approved and added to the repo.

**Review by richardlau — APPROVED** [review]

File: request-an-access-token.md [hunk 2]
```diff
@@ -34,9 +34,15 @@ The "secret name" is a string that the secret can be referenced in the GitHub Ac
...
-nodejs/import-in-the-middle | RELEASE_PLEASE_GITHUB_TOKEN
+The "expiration date" is the date before which the token should be renewed and
+replaced. This should be no longer than 1 year.
+
+The "pull request" is the PR that initially requested the token, or requested
+permission scope changes. The PR should describe the permission scopes requested.
+
+Repo                        | Secret name                  | Expiration date | Pull Request                              |
+---                         | ---                          | ---             | ---                                       |
+nodejs/import-in-the-middle | RELEASE_PLEASE_GITHUB_TOKEN  | 2025-07-23      | https://github.com/nodejs/admin/pull/902  |
```
Comment (aduh95) [inline request-an-access-token.md:45]:
```suggestion
[`nodejs/import-in-the-middle`][] | `RELEASE_PLEASE_GITHUB_TOKEN`  | 2025-07-23      | <https://github.com/nodejs/admin/pull/902> |
```

**Comment by legendecas** [comment]
Added a missing step found in https://github.com/nodejs/import-in-the-middle/issues/150 to grant @nodejs-github-bot write access on the repo if necessary.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**